specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2022-06-01-preview/loadtestservice.json

```diff
@@ -2032,7 +2032,7 @@
       "type": "object",
       "properties": {
         "value": {
-          "description": "The value of the secret. of type== akv-secret-uri or secret-value",
+          "description": "The value of the secret, of type== akv-secret-uri or secret-value",
```

Comment:
Can we add the label as it is? AKV_SECRET_URI, SECRET_VALUE. Will be better for customers that way.